src/iosource/pcap/Source.cc

```diff
@@ -176,7 +178,32 @@ void PcapSource::OpenOffline()
 	{
 	char errbuf[PCAP_ERRBUF_SIZE];
 
-	pd = pcap_open_offline(props.path.c_str(), errbuf);
+	FILE* f = nullptr;
+	if ( props.path == "-" )
+		{
+		f = stdin;
+		}
+	else
+		{
+		if ( ! (f = fopen(props.path.c_str(), "rb")) )
+			{
+			Error(util::fmt("unable to open %s: %s", props.path.c_str(), strerror(errno)));
+			return;
+			}
+
+		if ( BifConst::Pcap::bufsize_offline_bytes != 0 )
```

Comment:
I feel comment calling out that this implements the behavior where `bufsize_offline_bytes=0` would lead to use of the system default buffer size here would be helpful.